packages/cli/src/__snapshots__/run.test.ts.snap

```diff
@@ -22,27 +17,17 @@ exports[`Run command should execute "package start" and exit with 1: stderr 1`]
 
 exports[`Run command should execute "package start" and exit with 1: stdout 1`] = `""`;
 
-exports[`Run command should execute "package-one start" and exit with 0: stderr 1`] = `
-"warning package.json: No license field
```

Comment:
This error doesn't exist anymore in yarn berry - https://github.com/search?q=repo%3Ayarnpkg%2Fberry++%22No+license+field%22&type=code 

Was not fun to debug that 😆